### PR TITLE
uploaded open source modules

### PR DESCRIPTION
**Pull request by miifeanyi**

### CODE REVIEW

File: eks.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  token                  = data.aws_eks_cluster_auth.cluster.token
+  load_config_file       = false
+  version                = "~> 1.9"
+}
+
+module "my-cluster" {
+  source          = "terraform-aws-modules/eks/aws"
+  cluster_name    = "my-cluster"
+  cluster_version = "1.17"
+  subnets         = ["subnet-0cbf856b70ae278f6", "subnet-019016c386b1b9108", "subnet-022bd3c7456073ec1"]
```
Comment (vladimirmukhin) [inline eks.tf]: Subnets should not be hardcoded

File: eks.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  vpc_id          = "vpc-034305abe4ec968a8"
```
Comment (vladimirmukhin) [inline eks.tf]: VPC should not be hardcoded

File: kubeconfig_my-cluster [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+users:
+- name: eks_my-cluster
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1alpha1
+      command: aws-iam-authenticator
+      args:
+        - "token"
+        - "-i"
+        - "my-cluster"
```
Comment (vladimirmukhin) [inline kubeconfig_my-cluster]: Please don't commit kubeconfig to git

File: .terraform.lock.hcl [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    "zh:82f01abd905090f978b169ac85d7a5952322a5f0f460269dd981b3596652d304",
+    "zh:9a235610066e0f7e567e69c23a53327271a6fc568b06bf152d8fe6594749ed2b",
+    "zh:aeabdd8e633d143feb67c52248c85358951321e35b43943aeab577c005abd30a",
+    "zh:c20d22dba5c79731918e7192bc3d0b364d47e98a74f47d287e6cc66236bc0ed0",
+    "zh:c4fea2cb18c31ed7723deec5ebaff85d6795bb6b6ed3b954794af064d17a7f9f",
+    "zh:e21e88b6e7e55b9f29b046730d9928c65a4f181fd5f60a42f1cd41b46a0a938d",
+    "zh:eddb888a74dea348a0acdfee13a08875bacddde384bd9c28342a534269665568",
+    "zh:f46d5f1403b8d8dfafab9bdd7129d3080bb62a91ea726f477fd43560887b8c4a",
+  ]
+}
```
Comment (vladimirmukhin) [inline .terraform.lock.hcl]: Please add .terraform to gitignore

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: .terraform.lock.hcl (hunk 4 above)
Comment (miifeanyi) [reply .terraform.lock.hcl]: removed and added exclusion to .gitignore

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,1138 @@
...
+  description = "ARN of Outpost you want to create a subnet in."
+  type        = string
+  default     = null
+}
+
+variable "outpost_az" {
+  description = "AZ where Outpost is anchored."
+  type        = string
+  default     = null
+}
```
Comment (vladimirmukhin) [inline variables.tf]: variables.tf should be removed

Comment (miifeanyi) [reply variables.tf]: requested change made

File: kubeconfig_my-cluster (hunk 3 above)
Comment (miifeanyi) [reply kubeconfig_my-cluster]: kubeconfig file removed

File: eks.tf (hunk 2 above)
Comment (miifeanyi) [reply eks.tf]: change made

**Comment by miifeanyi** [comment]
image
Above is the error received after "terraform plan"

**Pull request merged**